test/e2e/framework/ingress_utils.go

```diff
@@ -378,6 +375,12 @@ func CleanupGCEIngressController(gceController *GCEIngressController) {
 	}
 }
 
+// CleanupGCEIngressController calls the GCEIngressController.Cleanup(false)
+// followed with deleting the static ip, and then a final GCEIngressController.Cleanup(true)
```

Comment:
Maybe move this comment to CleanupGCEIngressControllerWithTimeout since it makes more sense there?